odk1-src/launch-apps-from-collect.rst

```diff
@@ -21,7 +21,21 @@ Launching External Apps
 Launching external apps to populate single fields
 ------------------------------------------------------
 
-ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentString`` appearance. A ``value`` parameter that holds the current value for that field is passed to the application. Since v1.4.3, additional parameters can be specified. The names of these parameters are user defined and ``value`` is the only reserved name. 
+ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentAction(intentExtras)`` appearance.
+
+Under the covers, this feature configures an intent, a messaging objects used to request an action from another app component. For more details on intents,  refer to `these Android docs <https://developer.android.com/guide/components/intents-filters.html>`_.
+
+Collect uses the text immediately after ``ex:`` and before an opening parenthesis if it exists as the action the intent is built with. This action must be namespaced. That is, ``ex:org.opendatakit.counter`` is valid but ``ex:counter`` is not.
+
+The parameters defined in the optional parentheses represent extended data ("extras") to be added to the intent. The value to the left of the equals sign represents the extra name and may require a namespace. The value to the right of the equals sign represents the extra value. Multiple extras can be configured using a comma-separated list. The extra values can be:
```

Comment:
A suggestion...

```
Inside the optional parentheses you can define extended data which can be added to the intent. These are called `extras`_ and are represented by a comma-delimited list of ``name=value`` pairs. For example ``(please=think, of=examples, more=relevant, than=this)``. 

The values of extras can be:

- XPath expressions referring to other fields and including function calls
- String literals defined in single quotes
- Raw integers or decimals

```

Reply:
PS: 
Unlike markdown, bullet lists in RST should not be indented. Doing so will (sometimes) cause them to be wrapped in a blockquote.

```
- Bullets
- Like
- This

 - Not
 - Like
 - This
```